packages/expo-blur/src/BlurView.tsx

```diff
@@ -4,23 +4,17 @@ import { View, StyleSheet } from 'react-native';
 
 import { BlurViewProps } from './BlurView.types';
 
-type BlurViewForwardedRefProp = {
-  forwardedRef: React.ForwardedRef<View>;
-};
-
 const NativeBlurView = requireNativeViewManager('ExpoBlurView');
-class BlurView extends React.Component<BlurViewProps & BlurViewForwardedRefProp> {
+export default class BlurView extends React.Component<BlurViewProps> {
+  blurViewRef: React.Ref<typeof NativeBlurView> = null;
+
   /**
-   * This component is a composition of the two components, but from the outside it's
-   * just a simple View with additional properties. To properly handle `setNativeProps`
-   * method (used when animating props), we need to properly separate `ViewProps` from `BlurViewProps`
-   * and pass them to the proper underlying views.
-   *
-   * This method handles the native view reference obtained from the parent View component
-   * and overrides its original `setNativeProps` method that is available as its property.
-   * When the `NativeBlurView` native ref is available `BlurView`-only props are forwarded
-   * to this view using `setNativeProps` method exposed by the native runtime.
+   * When Animated.createAnimatedComponent(BlurView) is used Reanimated will detect and call this
+   * function to determine which component should be animated. We want to animate the NativeBlurView.
    */
+  getAnimatableRef() {
```

Comment:
I think we may need to mark as `@hidden` so that it doesn't show up in the docs (I don't think we need it to be exposed).